src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -299,12 +299,21 @@ private ExpressionSyntax Transform( ExpressionSyntax expr )
 			SimpleLambdaExpressionSyntax lambExpr => lambExpr
 				.WithModifiers( RemoveAsyncModifier( lambExpr.Modifiers ) )
 				.WithParameter( Transform( lambExpr.Parameter ) )
-				.WithExpressionBody( lambExpr.ExpressionBody != null ? Transform( lambExpr.ExpressionBody ) : null ),
+				.WithExpressionBody( lambExpr.ExpressionBody != null ? Transform( lambExpr.ExpressionBody ) : null )
+				.WithBlock( lambExpr.Block != null ? Transform( lambExpr.Block ) : null ),

```

Comment:
Let me know if you think I should move this change to another PR, it was a small miss